data/meta/ShipDef.lua

```diff
@@ -0,0 +1,44 @@
+-- Copyright © 2008-2024 Pioneer Developers. See AUTHORS.txt for details
+-- Licensed under the terms of the GPL v3. See licenses/GPL-3.txt
+
+-- This file implements type information about C++ modules for Lua static analysis
+
+---@meta
+
+---@alias ShipDef.Thrust { UP:number, DOWN:number, LEFT:number, RIGHT:number, FORWARD:number, REVERSE:number }
+
+---@class ShipDef
+---@field id string
+---@field name string
+---@field shipClass string
+---@field manufacturer string
+---@field modelName string
+---@field cockpitName string
+---@field tag ShipTypeTag
+---@field roles string[]
+--
+---@field angularThrust number
+---@field linearThrust ShipDef.Thrust
+---@field linAccelerationCap ShipDef.Thrust
+---@field effectiveExhaustVelocity number
+---@field thrusterFuelUse number -- deprecated
+---@field frontCrossSec number
+---@field sideCrossSec number
+---@field topCrossSec number
+---@field atmosphericPressureLimit number
+--
+---@field capacity number
```

Comment:
`capacity` seems to be a very generic term - what capacity is this?

Reply:
Equipment capacity. By convention, this is the name we have used for the amount of equipment volume the ship has available for 5+ years.

Reply:
I'd argue to rename it to `equipmentCapacity`, and this major rework is as good a time as any. Self-documenting code and all that.

But I don't want to break 5+ years of tradition..

Reply:
It's less tradition, and more that I'm not fully sure how much existing code uses the variable as-written. I'll see if it's a trivial change, but I don't really want to touch *more* files at the 11th hour.

Reply:
Feel free to ignore any suggestions I make...  certainly fully agree not to blow out the scope of this PR further.

Reply:
Globally renamed to `equipCapacity`. Ship.json files still define this key as `capacity`.